lib/openstack/openstack_event_monitor.rb

```diff
@@ -25,7 +28,7 @@ def self.plugin_priority
   # ordering of plugins.
   def self.subclasses
     # sort plugins on plugin_priorty
-    super.sort_by(&:plugin_priority)
+    (super - [OpenstackNullEventMonitor]).sort_by(&:plugin_priority)
```

Comment:
It's all part of my secret plan, @Fryguy 

I can change it back.  It was a wild thought that there was no sense in "testing" the null event monitor.  We test it, it fails (because it returns `false`), then we find that no event monitors matched (including the null event monitor), so we assign the null event monitor.

It's a "6-of-one" situation for me, though...